app/components/SuperAdminSideNavBar.tsx

```diff
@@ -0,0 +1,277 @@
+import { Link, useLocation } from "@remix-run/react";
+import React, { useState } from "react";
+
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "~/components/ui/accordion";
+
+const SuperAdminSideNavBar: React.FC = () => {
+  const [isSettingsOpen, setIsSettingsOpen] = useState(false);
+  const location = useLocation();
+
+  const mainMenuItems = [
+    {
+      name: "Dashboard",
+      icon: "/icons/dashboard.png",
+      path: "/admin/dashboard",
+    },
+    { name: "Products", icon: "/icons/products.png", path: "/admin/products" },
+    { name: "Users", icon: "/icons/users.png", path: "/admin/users" },
+    { name: "Email Template", icon: "/icons/email.png", path: "/admin/email" },
+    {
+      name: "Squeeze Pages",
+      icon: "/icons/squeeze.png",
+      path: "/admin/squeeze-pages",
+    },
+    {
+      name: "Waitlist Page",
+      icon: "/icons/waitlist.png",
+      path: "/admin/waitlist-page",
+    },
+    { name: "Settings", icon: "/icons/settings.png" },
+  ];
+
```

Comment:
move this to another file wthin the data folder:

settingsMenuItems, organizationItem and mainMenuItems